scripts/perf/perf_confg.template.json

```diff
@@ -0,0 +1,16 @@
+{
+  "main_node_dir": "absolute/path/to/saito-rust/saito-rust",
+  "spammer_node_dir": "/absolute/path/to//saito-spammer",
```

Comment:
should contain the ip address and the port instead of the directory. can ssh to get the stat file, block file info later. ok for now for initial phase